tensorflow_datasets/core/download/download_manager.py

```diff
@@ -514,25 +515,32 @@ def callback(path):
       return self._extract(resource)
     return self._download(resource).then(callback)
 
-  def download_kaggle_data(self, dataset_name: str):
+  def download_kaggle_data(self, competition: str):
     """Download data for a given Kaggle Dataset or competition.
 
     Note: This function requires Kaggle CLI tool. Read the installation guide
     at https://www.kaggle.com/docs/api.
 
     Args:
-      dataset_name: Dataset name (`zillow/zecon`) or competition name
-        (`titanic`)
+      competition: Dataset name (`zillow/zecon`) or competition name (`titanic`)
 
     Returns:
       The path to the downloaded files.
     """
-    with self._downloader.tqdm():
-      kaggle_downloader = self._downloader.kaggle_downloader(dataset_name)
-      urls = kaggle_downloader.competition_urls
-      files = kaggle_downloader.competition_files
-      return _map_promise(self._download,
-                          dict((f, u) for (f, u) in zip(files, urls)))
+    kaggle_downloader = kaggle.KaggleCompetitionDownloader(competition)
```

Comment:
The kaggle logic should stay in kaggle. I was proposing to move all this code in `kaggle.py` and only call `kaggle.download_kaggle_data(competition, output_dir)`

Reply:
Fixed